docs/source/api/auth.md

```diff
@@ -24,10 +24,12 @@
 
 ```{eval-rst}
 .. autoconfigurable:: PAMAuthenticator
+   :members:
```

Comment:
Previews:
- This PR: https://jupyterhub--4253.org.readthedocs.build/en/4253/api/auth.html#pamauthenticator
- Main branch: https://jupyterhub.readthedocs.io/en/latest/api/auth.html#pamauthenticator